src/replication/ng/policy/manager/manager.go

```diff
@@ -156,25 +156,20 @@ func (m *DefaultManager) Create(policy *model.Policy) (int64, error) {
 
 // List returns all the policies
 func (m *DefaultManager) List(queries ...*model.PolicyQuery) (total int64, policies []*model.Policy, err error) {
-	// default query parameters
-	var name = ""
-	var namespace = ""
-	var page int64 = 1
-	var pageSize int64 = 15
-
-	if len(queries) > 0 {
-		name = queries[0].Name
-		namespace = queries[0].Namespace
-		page = queries[0].Pagination.Page
-		pageSize = queries[0].Pagination.Size
+	// Generate policy query parameters, if pagination not set, set to default value.
+	query := &model.PolicyQuery{}
```

Comment:
I don't think the default values for pagination are necessary as we have handle it in the API level. And we do need to list all policies  by some other internal component, the default values for pagination will make it complicated.

Reply:
Updated